tsconfig.json

```diff
@@ -2,7 +2,6 @@
     "extends": "./tsconfig-base.json",
     "compilerOptions": {
         "target": "ES6",
-        "module": "ES6",
-        "outDir": "build/esm",
+        "module": "CommonJS",
```

Comment:
Does this not cause issues for the ui? Fwiw I had this
```diff
--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,8 +1,15 @@
 {
     "extends": "./tsconfig-base.json",
+    "ts-node": {
+        // these options are overrides used only by ts-node
+        // same as the --compilerOptions flag and the TS_NODE_COMPILER_OPTIONS environment variable
+        "compilerOptions": {
+          "module": "commonjs"
+        }
+      },
```

Reply:
See pr description

Reply:
I think my version is mergable but yea I guess we don't want the register-asset script